include/flatbuffers/idl.h

```diff
@@ -558,6 +561,7 @@ struct IDLOptions {
   std::string cpp_object_api_pointer_type;
   std::string cpp_object_api_string_type;
   bool cpp_object_api_string_flexible_constructor;
+  int cpp_object_api_field_case;
```

Comment:
Why this is `int` instead of `Case`.

Reply:
will now be CaseStyle enum (renamed from Case) with next commit